tslint.json

```diff
@@ -1,60 +1,29 @@
 {
   "extends": [
-    "tslint-react",
     "tslint-config-prettier"
   ],
   "rulesDirectory": [
+    "node_modules/tslint-react/rules",
     "node_modules/vrsource-tslint-rules/rules",
     "node_modules/tslint-microsoft-contrib/",
     "tslint-rules/"
   ],
   "rules": {
     "button-group-order": true,
     "class-name": true,
-    "curly": true,
     "indent": [
       true,
       "spaces"
     ],
-    "insecure-random": [ true ],
-    "interface-name": [ true, "always-prefix"],
     "jsdoc-format": true,
-    "literal-spacing": [
-      false,
-      {
-        "array": ["never"],
-        "object": ["always"]
-      }
-    ],
-    "member-access": [
-      true,
-      "check-accessor",
-      "check-constructor"
-    ],
     "member-ordering": [
       true,
       "static-before-instance",
       "variables-before-functions"
```

Comment:
Both of these options are deprecated. Does the [default order](https://github.com/nzakas/eslint-plugin-typescript/blob/master/docs/rules/member-ordering.md#options) in the ESLint plugin look OK?